upup/models/vfs.go

```diff
@@ -17,17 +17,21 @@ limitations under the License.
 package models
 
 import (
+	"embed"
 	"errors"
 	"io"
+	"io/fs"
```

Comment:
Unrelated, but maybe worth looking at whether we can use more of `io/fs` in our vfs layer :-)